build/com_google_private_join_and_compute/repo.bzl

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 The Cross-Media Measurement Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Repository rules/macros for private-join-and-compute library.
+
+See https://github.com/google/private-join-and-compute
+"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def private_join_and_compute_repo(commit, sha256):
```

Comment:
This doesn't look like it's called... probably because it's accidentally not called in any of the other repos either. See https://github.com/world-federation-of-advertisers/cross-media-measurement/pull/30/files for how we're fixing this in the other repo.